larray/io/excel.py

```diff
@@ -321,10 +321,26 @@ def __setitem__(self, key, value):
         @property
         def shape(self):
             # include top-left empty rows/columns
-            # XXX: is there an exposed xlwings API for this? expand maybe?
+            # see https://www.excelcampus.com/vba/find-last-row-column-cell/#endcode
```

Comment:
unsure the link is still relevant?

Reply:
Since we do not implement the exact same logic, I would rather have the link in the issue for posterity than in the code.